bundles/org.openhab.binding.miele/src/main/java/org/openhab/binding/miele/internal/handler/CoffeeMachineHandler.java

```diff
@@ -14,6 +14,8 @@
 
 import static org.openhab.binding.miele.internal.MieleBindingConstants.APPLIANCE_ID;
 
+import com.google.gson.JsonElement;
```

Comment:
Can you put all imports back in order. Any other person who would use the Eclipse IDE would otherwise change get diffs by just saving the file. Eclipse will reorder the imports according to our coding style.